lib/bolt/util.rb

```diff
@@ -78,11 +78,23 @@ def read_optional_yaml_hash(path, file_name)
       end
 
       def first_runs_free
+        # If this fails, use the system path instead
+        FileUtils.mkdir_p(Bolt::Config.user_path)
         Bolt::Config.user_path + '.first_runs_free'
+      rescue StandardError
+        begin
+          # If using the system path fails, then don't bother with the welcome
+          # message
+          FileUtils.mkdir_p(Bolt::Config.system_path)
+          Bolt::Config.system_path + '.first_runs_free'
```

Comment:
Since it's fairly unlikely that a user will be running with permissions to write to this directory, I think it might make sense to not fall back to the system path if the user path fails.

Reply:
I don't see why we wouldn't at least try, and give up if it's not writable? This is the same fallback we have for default projects, which is essentially what I'm trying to do here - write to the default project without actually loading config. Is there a risk you're concerned about?